tasks.py.jinja

```diff
@@ -309,6 +354,115 @@ def start(c, detach=True, debugpy=False):
     _logger.info("Waiting for services to spin up...")
     time.sleep(SERVICES_WAIT_TIME)
 
+@task(
+    develop,
+    help={
+        "modules": "Comma-separated list of modules to test. Default: 'base'.",
+        "core": "Install all core addons. Default: False",
+        "extra": "Install all extra addons. Default: False",
+        "private": "Install all private addons. Default: False",
+    },
+)
+def install(c, modules=None, core=False, extra=False, private=False):
+    """Install Odoo addons"""
+    if not (modules or core or extra or private):
+        raise exceptions.ParseError(
+            message="You must provide at least one option. "
+            "See --help for details."
+        )
+    cmd = "docker-compose run --rm addons init"
+    if core:
+        cmd += " --core"
+    if extra:
+        cmd += " --extra"
+    if private:
+        cmd += " --private"
+    if modules:
+        cmd += f" -w {modules}"
+    with c.cd(str(PROJECT_ROOT)):
+        c.run(
+            cmd,
+            env=dict(
+                UID_ENV,
+            ),
+        )
+
+
+def _get_cwd_addon(file=None):
+    if not file:
+        cwd = Path.cwd()
+    else:
+        cwd = Path.parent(file)
```

Comment:
According to your loop below, this is the same but simpler:

```suggestion
def _get_cwd_addon(file="."):
    cwd = Path(file)
```

Reply:
Actually, I need the **parent** if you pass a file, and the cwd if not. So I think it is better to do this verification. OTOH, I simplified the line to:
```
cwd = Path(file).parent if file else Path.cwd()
```